Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM mhart/alpine-node:5.6.0
+
+# Install required dependencies (Alpine Linux packages)
+RUN apk update && \
+  apk add --no-cache \
+    g++ \
+    gcc \
+    git \
+    libev-dev \
+    libevent-dev \
+    libuv-dev \
+    make \
+    openssl-dev \
+    perl \
+    python
+
+# Switch to directory for external dependencies (installed from source)
+WORKDIR /external
+
+# Install (global) NPM packages/dependencies
+RUN npm install -g \
+  node-gyp
+
+# Switch to directory with sources
+WORKDIR /src
+
+# Copy package json and install dependencies
+COPY package.json .
+
+# Install (local) NPM and Bower packages/dependencies
+RUN npm install
```

Comment:
This will install all dependencies, including the `devDependencies` section. What do you think about setting `NODE_ENV=production` to prevent bloating the image?